pkg/vgmanager/vgmanager_controller.go

```diff
@@ -488,15 +609,52 @@ func (r *VGReconciler) generateVolumeGroupNodeStatus() (*lvmv1alpha1.LVMVolumeGr
 	//lvmvgstatus := vgNodeStatus.Spec.LVMVGStatus
 	var statusarr []lvmv1alpha1.VGStatus
 
+	var vgExists bool
+
 	for _, vg := range vgs {
+		// Add pvs as per volumeGroup CR if path is given add path else add name
+		diskPattern := internal.DiskByNamePrefix
+
+		lvmVolumeGroup, ok := lvmVolumeGroups[vg.Name]
+		if !ok {
+			continue
+		}
+
+		deviceSelector := lvmVolumeGroup.Spec.DeviceSelector
+		if deviceSelector != nil && len(deviceSelector.Paths) > 0 {
+			if filepath.Dir(deviceSelector.Paths[0]) == internal.DiskByPathPrefix {
+				diskPattern = internal.DiskByPathPrefix
+			}
+		}
+
+		devices := []string{}
+		if diskPattern == internal.DiskByPathPrefix {
```

Comment:
idea: why not always use the disk path in the status?!

Reply:
I had this discussion earlier with @nbalacha and we came to the conclusion that we will show the paths as given in the spec. It will be easier for a user to compare them from the spec.